src/test/scala/org/zalando/nakadi/client/KlientSpec.scala

```diff
@@ -74,44 +44,28 @@ class KlientSpec extends WordSpec with Matchers with BeforeAndAfterEach with Laz
     }
   }
 
-  private def performStandardRequestChecks(expectedRequestPath: String, expectedRequestMethod: HttpString) = {
+  private def performStandardRequestChecks(expectedRequestPath: String, expectedRequestMethod: HttpString): NakadiRequest = {
```

Comment:
Providing the return value instead of relying on type inference is important for two reasons:
- to inform the human reader on what to expect, without needing to read the body
- to encapsulate compile errors e.g. deriving of refactoring. Without giving the return type, changing the function will cause changes to ripple to its callers, messing up their syntax highlighting (and more serious stuff).